### PR TITLE
Persist Interactive window visibility across VS restarts

### PR DESCRIPTION
**Pull request by amcasey**
Set ```__VSCREATETOOLWIN.CTW_fForceCreate``` as in
https://github.com/Microsoft/PTVS/blob/21e5600e090143ced879e60da4e88bf63b6b6f99/Python/Product/PythonTools/PythonTools/Repl/InteractiveWindowProvider.cs

### CODE REVIEW

**Comment by amcasey** [comment]
FYI @tmat @ManishJayaswal @KevinH-MS @cston

**Comment by amcasey** [comment]
@pharring Can you please have a look?

**Comment by KevinH-MS** [comment]
:+1:

**Comment by pharring** [comment]
:+1:

File: src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowProvider.cs [hunk 1]
```diff
@@ -79,7 +80,8 @@ public IVsInteractiveWindow Create(int instanceId)
         {
             var evaluator = CreateInteractiveEvaluator(_vsServiceProvider, _classifierAggregator, _contentTypeRegistry, _vsWorkspace);
 
-            var vsWindow = _vsInteractiveWindowFactory.Create(Id, instanceId, Title, evaluator, 0);
+            // ForceCreate means that the window should be created if the persisted layout indicates that it is visible.
```
Comment (ManishJayaswal) [inline src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowProvider.cs]: What does visible mean here? If it is open but the interactive window tab is not selected then is it considered visible? If yes then this seems right.

Comment (amcasey) [inline src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowProvider.cs]: The foreground/background status of the Interactive window tab appears to be persisted correctly.

**Comment by amcasey** [comment]
Test this please.

**Pull request merged**